ro.go.hmlendea.Duolingo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: ro.go.hmlendea.Duolingo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+separate-locales: false
+command: duolingo-desktop
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: duolingo-desktop
+    buildsystem: simple
+    build-commands:
+      - unzip -d duolingo-desktop duolingo-desktop_*.zip
+      - cp -r duolingo-desktop /app
+      - install -Dm 644 icon.svg ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/duolingo.svg
```

Comment:
```suggestion
      - install -Dm 644 icon.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/duolingo.svg
```

Reply:
I've switched to the icon from upstream, which is a png, so I'll keep using the 512x512 dir